### PR TITLE
add sentry

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/helpers/handleError.ts [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+}: {
+  e: unknown
+  sendToSentry?: boolean
+}) {
+  console.error('Handled error\n', e)
+
+  const message = parseError(e)
+  if (sendToSentry) {
+    if (e instanceof Error) {
+      console.log('catch')
```
Comment (T-Damer) [inline src/helpers/handleError.ts]: We don't need it

File: src/helpers/initSentry.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import * as Sentry from '@sentry/react'
+import env from 'helpers/env'
+
+export function initSentry() {
+  Sentry.init({
+    dsn: env.VITE_SENTRY_DSN,
+    environment: process.env.NODE_ENV,
```
Comment (T-Damer) [inline src/helpers/initSentry.ts:7]: I think we don't have it in vite/browser, make sure it logs dev/prod correctly

File: src/helpers/parseError.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { isAxiosError } from 'axios'
+
+interface MessageObject {
+  message: string
+}
+
+export function stringifyError(e: unknown) {
+  if (typeof e === 'undefined') return e
```
Comment (T-Damer) [inline src/helpers/parseError.ts]: Should return something else

Comment (MixailE) [reply src/helpers/parseError.ts]: Just copied from https://github.com/BigWhaleLabs/ketl-invites-frontend/blob/main/src/helpers/parseError.ts

File: src/helpers/initSentry.ts (hunk 2 above)
Comment (MixailE) [reply src/helpers/initSentry.ts:7]: Works fine with https://github.com/BigWhaleLabs/ketl-invites-frontend/blob/6b9c636e9d6c81e4820828dc873194d3a89416f4/src/helpers/initSentry.ts#L7

File: src/helpers/handleError.ts (hunk 1 above)
Comment (MixailE) [reply src/helpers/handleError.ts]: removed

File: src/helpers/env.ts [hunk 4]
```diff
@@ -19,5 +19,6 @@ export default cleanEnv(import.meta.env, {
   VITE_KETL_INVITES_BACKEND: str({
     default: KETL_INVITES_BACKEND,
   }),
+  VITE_SENTRY_DSN: str(),
```
Comment (smyzin) [inline src/helpers/env.ts:22]: Don't forget to update env file in the 1Pass

File: src/helpers/parseError.ts (hunk 3 above)
Comment (smyzin) [reply src/helpers/parseError.ts]: If `undefined` we should provide some default text/error, I think

**Review by smyzin — APPROVED** [review]

File: src/helpers/env.ts (hunk 4 above)
Comment (MixailE) [reply src/helpers/env.ts:22]: Ok

**Pull request merged**